### PR TITLE
Add a bitcoin network node

### PR DESCRIPTION
**Pull request by dangershony**

### CODE REVIEW

File: BTC/src/Bitcoin.Node/Bitcoin.Node.csproj [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+   <ItemGroup>
+      <PackageReference Include="Blockcore.Features.BlockStore" Version="1.0.*" />
+      <PackageReference Include="Blockcore.Features.Consensus" Version="1.0.*" />
+      <PackageReference Include="Blockcore.Features.MemoryPool" Version="1.0.*" />
+      <PackageReference Include="Blockcore.Features.Miner" Version="1.0.*" />
+      <PackageReference Include="Blockcore.Features.NodeHost" Version="1.0.*" />
+      <PackageReference Include="Blockcore.Features.RPC" Version="1.0.*" />
+      <PackageReference Include="Blockcore.Features.Wallet" Version="1.0.*" />
+      <PackageReference Include="Blockcore.NBitcoin" Version="1.0.*" />
+      <PackageReference Include="Blockcore.Networks.Bitcoin" Version="1.0.13" />
```
Comment (sondreb) [inline BTC/src/Bitcoin.Node/Bitcoin.Node.csproj]: Should not be specific.

Comment (sondreb) [reply BTC/src/Bitcoin.Node/Bitcoin.Node.csproj]: I fixed this

**Review by sondreb — APPROVED** [review]
Looks good, have verified I get connections and sync starts.

**Comment by sondreb** [comment]
Great works, thanks!

File: BTC/src/Bitcoin.Node/Bitcoin.Node.csproj (hunk 1 above)
Comment (dangershony) [reply BTC/src/Bitcoin.Node/Bitcoin.Node.csproj]: ah yeah good spot

**Pull request merged**